tests/run_tests.py

```diff
@@ -351,28 +357,37 @@ def run_system(config_file, vic_exe, test_data_dir, out_dir, driver):
                                                         expectation))
 
             # -------------------------------------------------------- #
-            # check output files
-            if test_dict['check']:
-                if 'complete' in test_dict['check']:
-                    start = None
-                    end = None
-
-                for fname in glob.glob(os.path.join(dirs['results'], '*')):
-                    df = read_vic_ascii(fname, header=True)
-                    # do numeric tests
-
-                    # check that each dataframe includes all timestamps
+            # check output files (different tests depending on driver)
+            if driver == "classic":
+                if test_dict['check']:
                     if 'complete' in test_dict['check']:
-                        if (start is not None) and (end is not None):
-                            check_completed(df, start, end)
-                        else:
-                            start = df.index[0]
-                            end = df.index[-1]
-
+                        start = None
+                        end = None
+
+                    for fname in glob.glob(os.path.join(dirs['results'], '*')):
+                        df = read_vic_ascii(fname, header=True)
+                        # do numeric tests
+
+                        # check that each dataframe includes all timestamps
+                        if 'complete' in test_dict['check']:
+                            if (start is not None) and (end is not None):
+                                check_completed(df, start, end)
+                            else:
+                                start = df.index[0]
+                                end = df.index[-1]
+            elif driver == "image": 
+                if test_dict['check']:
                     # check for nans in the df
                     if 'nonans' in test_dict['check']:
                         check_for_nans(df)
-            # -------------------------------------------------------- #
+            
+                    # check for nans in domain file  
+                    if 'output_nans' in test_dict['check']:
+                        print(test_dict['domain_file'])
+                        ds_domain = xr.open_dataset(os.path.join(test_data_dir + 'image/Stehekin/parameters/',
+                                                    test_dict['domain_file']))
+                        ds_output = xr.open_dataset(os.path.join(dirs['results'], 'Stehekin.history.nc')) 
```

Comment:
We'll want to remove these hard coded paths and get everything from the test config file.